docs/api/browser-window.md

```diff
@@ -983,6 +983,29 @@ Same as `webContents.capturePage([rect, ]callback)`.
   * `httpReferrer` String - A HTTP Referrer url.
   * `userAgent` String - A user agent originating the request.
   * `extraHeaders` String - Extra headers separated by "\n"
+  * `postData` Array - An array of `upload` objects which
```

Comment:
Although not yet merged, the standard for arrays of structured objects will be set in #7379

So this would preferably be

 ```markdown
  * `postData` [UploadData[]](structures/upload-data.md) - An array of `upload` objects
```

With the `UploadData` structure specified in a parseable format (I.e. We will have to be more crafty around the if statements)

/cc @zeke Any ideas on how to handle these if statements.  I think making four separate structs `UploadData`, `UploadFile`, `UploadFileSystem` and `UploadBlob`